checks/zookeeper_test.go

```diff
@@ -1,7 +1,8 @@
-package checks
+package checks_test
```

Comment:
These do need to be in the "checks" package. From [the golang documentation](https://golang.org/pkg/testing/),

> To write a new test suite, create a file whose name ends _test.go that contains the TestXxx functions as described here. Put the file in the same package as the one being tested. The file will be excluded from regular package builds but will be included when the “go test” command is run. For more detail, run “go help test” and “go help testflag”.